AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_constants.py

```diff
@@ -139,11 +139,13 @@ class AtomComponentProperties:
     def actor(property: str = 'name') -> str:
         """
         Actor component properties.
+          - 'Actor asset' Asset.id of the hair TressFX asset.
```

Comment:
is this always a "hair TressFX asset"? I only ask because the name 'Actor asset' makes it seem like it could be more than just hair

Reply:
copy paste error. I'll fix that